### DISCUSSION TITLE
[POC] New Docs stack using mkdocs

### DISCUSSION THREAD

**Discussion by agardnerIT** [perses/perses#2210] [Show and tell]
Hi all, I spent some time reworking the docs to use mkdocs. Curious on your thoughts.

<img width="1241" alt="Screenshot 2024-08-30 at 9 27 06 AM" src="https://github.com/user-attachments/assets/94ffa0a7-6107-462c-83ac-db8dc1458064">

See here: https://github.com/perses/website/issues/33

[As I've already stated](https://github.com/perses/website/issues/31#issuecomment-2308774433) I'm happy to maintain this stack and contribute to the docs.

**Comment by Nexucis** [comment]
yeah that would be great to have someone that is eager to improve and maintain the doc and the website !

Closing this discussion in favor of the issue https://github.com/perses/website/issues/33 where we are currently discussing about what `mkdocs` can provide and improve comparing to the current situation.

Thank you for proposing your help ! 🙏
